### ISSUE TITLE
WQE with Failed status

### ISSUE THREAD

**Issue by vlimant** [dmwm/WMCore#7910]
https://cmsweb.cern.ch/couchdb/workqueue/_design/WorkQueue/_rewrite/elementsInfo?request=pdmvserv_task_HIN-pPb816Spring16GS-00048__v1_T_170601_154923_3839

has 

https://cmsweb.cern.ch/couchdb/workqueue/_design/WorkQueue/_rewrite/element/a98e9f4b94b7913d325f861d315ed59f

 in Failed

{"_id":"a98e9f4b94b7913d325f861d315ed59f","_rev":"3-986e80c0c10b87faeeae6a719451d894","updatetime":1496343657.3619999886,"WMCore.WorkQueue.DataStructs.WorkQueueElement.WorkQueueElement":{"ParentQueueId":"pdmvserv_task_HIN-pPb816Spring16GS-00048__v1_T_170601_154923_3839","NumberOfEvents":20000000,"StartPolicy":"MonteCarlo","CreationTime":1496343427.4845480919,"NumOfFilesAdded":0,"Priority":85000,"ParentData":{},"PileupData":{},"RejectedInputs":[],"ParentFlag":false,"TaskName":"HIN-pPb816Spring16GS-00048_0","Status":"Failed","Inputs":{},"NumberOfLumis":200005,"Jobs":37,"ParentQueueUrl":"https://cmsweb.cern.ch/couchdb/workqueue","ChildQueueUrl":"http://vocms0304.cern.ch:5984","PercentSuccess":0,"PercentComplete":0,"WMBSUrl":null,"OpenForNewData":false,"EndPolicy":{"policyName":"SingleShot"},"FilesProcessed":0,"SiteWhitelist":["T2_DE_DESY","T2_TR_METU","T2_AT_Vienna","T2_CH_CSCS","T2_TW_NCHC","T2_UK_SGrid_RALPP","T2_FR_GRIF_LLR","T2_BE_UCL","T2_ES_IFCA","T2_DE_RWTH","T2_UK_SGrid_Bristol","T2_FR_IPHC","T2_KR_KNU","T1_ES_PIC","T1_UK_RAL","T2_IT_Legnaro","T2_CN_Beijing","T2_UK_London_Brunel","T2_RU_JINR","T2_GR_Ioannina","T2_FR_CCIN2P3","T2_CH_CERN_HLT","T1_FR_CCIN2P3","T2_FR_GRIF_IRFU","T3_IN_TIFRCloud","T2_UK_London_IC","T2_IT_Bari","T2_IT_Rome","T1_DE_KIT","T2_ES_CIEMAT","T1_RU_JINR","T2_HU_Budapest","T2_US_MIT","T2_BE_IIHE","T2_CH_CERN","T2_PT_NCG_Lisbon","T2_PL_Swierk","T2_BR_SPRACE"],"ACDC":{},"NumberOfFiles":0,"TeamName":"production","NoPileupUpdate":false,"RequestName":"pdmvserv_task_HIN-pPb816Spring16GS-00048__v1_T_170601_154923_3839","SubscriptionId":null,"NoInputUpdate":false,"blowupFactor":1,"TimestampFoundNewData":0,"ProcessedInputs":[],"SiteBlacklist":[],"Mask":{"LastRun":1,"FirstRun":1,"inclusivemask":true,"runAndLumis":{},"LastEvent":20000000,"FirstEvent":1,"LastLumi":200005,"FirstLumi":1},"Dbs":null,"EventsWritten":0},"timestamp":1496343427.4845480919,"thunker_encoded_json":true,"type":"WMCore.WorkQueue.DataStructs.WorkQueueElement.WorkQueueElement"}

any specific reason ?

Where could we expose that reason ?

**Comment by amaltaro** [comment]
I don't really know what happened here and what - if I'm right - that's not how it was supposed to happen either...

It's a MonteCarlo policy with `"NumberOfLumis":200005`, thus exceeding the usual limit we had in global workqueue, which was changed 10 days ago to 400k:
https://github.com/dmwm/WMCore/issues/7855

This was supposed to be logged into the LogDB (couch, but I see no request documents up there).

Can you clone/assign it and give us the request name please?

**Comment by amaltaro** [comment]
BTW, besides the cleanUpTask thread with the following log over and over again:
```
2017-06-01 21:02:56,230:INFO:WorkQueue:Request pdmvserv_task_HIN-pPb816Spring16GS-00048__v1_T_170601_154923_3839 finished ({'Status': 'Failed', 'PercentSuccess': 0, 'PercentComplete': 0})
```
there is no other log mentioned this WQE/request being failed.

**Comment by amaltaro** [comment]
Wait!!! Here is the reason... sigh... we also have to update this setting on every single agent, because the agent uses two workqueue couch databases and, when it writes to the local `workqueue` database, it also goes through the start policy algorithms and it again validates the number of lumi sections in WQE.

I think there are two issues here:
1. agents should not be validating this lumi limit
2. actually, I think it shouldn't even call those start policy classes when creating a new WQE for the local workqueue db.

In summary, a plain clone is not going to work. I suggest we close this issue and follow this up on https://github.com/dmwm/WMCore/issues/7855

**Comment by amaltaro** [comment]
Patch was pushed in into all the agents yesterday afternoon.
